tree/dataframe/inc/ROOT/RResultPtr.hxx

```diff
@@ -170,7 +170,15 @@ public:
 
    /// Get a const reference to the encapsulated object.
    /// Triggers event loop and execution of all actions booked in the associated RLoopManager.
-   const T &GetValue() { return *Get(); }
+   const T &GetValue()
+   {
+       try{
+           if( fObjPtr != nullptr)
+               throw fObjPtr;
```

Comment:
in C++ you don't throw objects, this should throw a `std::runtime_error`. Also we shouldn't try-catch our own throw, and the error condition is `fObjPtr == nullptr`

Reply:
Since you had mentioned about throwing exception, can you suggest what try catch can be replaced with.

Reply:
Sure, not problem!

Library code just throws the error. Client code can catch if it wants. I.e. there should be no try/catch here, just the throw

Reply:
@eguiraud Review Changes Completed
Thank you !